package.json

```diff
@@ -13,47 +13,47 @@
   "license": "MIT",
   "homepage": "https://github.com/planetary/airframe",
   "engines": {
-    "node": ">=4.2.0",
-    "npm": ">=2.1.18"
+    "node": ">=6.9.0",
+    "npm": ">=3.10.8"
   },
   "scripts": {
     "prebuild": "rimraf build",
-    "build": "cross-env NODE_ENV=prod webpack -p",
+    "build": "webpack --env.prod -p",
     "postbuild": "imagemin assets/images/* --out-dir=build/images",
     "prebuild:docs": "rimraf build",
-    "build:docs": "cross-env NODE_ENV=prod WITH_DOCS=true webpack -p",
+    "build:docs": "webpack --env.prod --env.docs -p",
     "postbuild:docs": "imagemin assets/images/* --out-dir=build/images",
     "prebuild:dev": "rimraf build",
-    "build:dev": "cross-env NODE_ENV=dev MODE=build webpack",
+    "build:dev": "webpack --env.dev --env.build",
     "postbuild:dev": "imagemin assets/images/* --out-dir=build/images",
-    "dev": "cross-env NODE_ENV=dev webpack-dev-server --inline --hot",
+    "dev": "webpack-dev-server --inline --hot --env.dev",
     "validate": "npm-run-all --parallel lint build",
     "lint": "eslint assets"
   },
-  "dependencies": {
+  "devDependencies": {
```

Comment:
That's a good Q. If devDependencies can't be installed on deployment servers then I guess we would have to make them regular dependencies.

I changed it to this because these are all development dependencies, and I was unaware of that potential limitation. 

Let me know and I'll update